### PR TITLE
SPCTR-3455 Fix: Global - Resolved an issue involving a block encounter error when attempting to select typography font families, stemming from the presence of multiple iframes on the page.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3455

### Screenshots

- https://d.pr/v/x16HKg

### Types of changes

- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop Google map.
- After that use heading block.
- Go to style, Typography and select 2 or 3 fonts family
- Make sure you did not get block encounter an error issue.
- Switch to responsive devices and select typography font family again.
- Make sure you did not get block encounter an error issue

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/typography/fontloader.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const WebfontLoader = ( props ) => {
 			addFont( props.config.google.families[ 0 ] );
 		}
 
-		let iframeFound = document.getElementsByTagName( 'iframe' );
+		let iframeFound = document.getElementsByName( 'editor-canvas' );
```
Comment (rahulvarma722) [inline src/components/typography/fontloader.js:63]: For enhanced code reliability, I recommend using `document.querySelector('iframe[name="editor-canvas"]')` instead of `document.getElementsByName('editor-canvas')`. This change ensures that we specifically target the iframe element with the name attribute set to 'editor-canvas.' By using `document.getElementsByName('editor-canvas')`, there's a risk of unintended consequences if another HTML node with the name 'editor-canvas' exists elsewhere in the document. Employing `querySelector` with the precise selector minimizes the chance of conflicts and potential functionality issues.

Comment (AbhijitNage123) [reply src/components/typography/fontloader.js]: Done as suggested. Working as expected. Thanks @rahulvarma722

**Review by rahulvarma722 — APPROVED** [review]

File: src/components/typography/fontloader.js [hunk 2]
```diff
@@ -73,7 +70,7 @@ const WebfontLoader = ( props ) => {
 				inactive: handleInactive,
 				context: iframeFound.contentWindow,
 			} );
-			addFont( props.config.google.families[ 0 ] );
+			addFont( props.config.google.families[0] );
```
Comment (adi3890) [inline src/components/typography/fontloader.js:73]: @AbhijitNage123 why this change.
Looks inconsistent.

Comment (AbhijitNage123) [reply src/components/typography/fontloader.js]: Done

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]
